aicsimageio/aics_image.py

```diff
@@ -221,7 +370,7 @@ def get_image_dask_data(
         ----------
         dimension_order_out: Optional[str]
             A string containing the dimension ordering desired for the returned ndarray.
-            Default: "TCZYX"
+            Default: dimensions.DEFAULT_DIMENSION_ORDER (with or without Samples)
```

Comment:
In the context for Samples for RGB/BGR images do we intend to allow Sample=1 to say pull out only the green channel?

Reply:
I haven't tested it but I think it simply natively supports this.

```python
img  # has dims "TCZYXS"
img.get_image_data("ZYX", S=2)  # pulls the B from RGB samples (along with 0 for the other dimensions)
```

Reply:
I will add a test to ensure it's possible.

Reply:
Added test to `test_transforms` with a specific example of this behavior. It all works natively.